test/apis/stacks.test.js

```diff
@@ -56,6 +55,47 @@ test('stacks.create', t => {
   td.verify(requestStub(td.matchers.contains(expectedArgs), td.matchers.anything()))
 })
 
+test('stacks.create', t => {
+  const rokka = rka({ apiKey: 'APIKEY' })
+  const options = {'jpg.quality': 76}
+  const operations = [
+    rokka.operations.rotate(45),
+    rokka.operations.resize(100, 100)
+  ]
+
+  rokka.stacks.create('myorg', 'mystack', {operations: operations, options: options})
+
+  const expectedArgs = {
+    method: 'PUT',
+    uri: 'https://api.rokka.io/stacks/myorg/mystack',
+    body: {operations, options},
+    qs: {}
+  }
+
+  td.verify(requestStub(td.matchers.contains(expectedArgs), td.matchers.anything()))
+})
+
+test('stacks.create (with expressions)', t => {
+  const rokka = rka({ apiKey: 'APIKEY' })
+  const options = {'jpg.quality': 76}
+  const operations = [
+    rokka.operations.rotate(45),
+    rokka.operations.resize(100, 100)
+  ]
+  const expressions = [
+    rokka.expressions.default('options.dpr >= 2', {'jpg.quality': 60, 'webp.quality': 60})
+  ]
+  rokka.stacks.create('myorg', 'mystack', {operations: operations, options: options, expressions: expressions})
+  const expectedArgs = {
+    method: 'PUT',
+    uri: 'https://api.rokka.io/stacks/myorg/mystack',
+    body: {operations: operations, options: options, expressions: expressions},
+    qs: {}
+  }
```

Comment:
also here the two object arguments can be simplified as:
```
{ operations, options, expressions }
```

as it's the same as the body I'd put that into a variable.